tests/DCPS/XTypes/run_test.pl

```diff
@@ -100,7 +100,7 @@ sub run_test {
 if ($test_name eq '') {
   while (my ($k, $v) = each %params) {
     run_test ($v, $k);
-    sleep(7);
+    sleep(10);
```

Comment:
Why is any amount of delay needed here?

Reply:
The readers' wait sets start timing out if "too many" tests are running simultaneously. Locally 7s works, but Katydid shows this symptom with 7s. Fred and I discussed it and decided to try increasing it to see if it would help. There is an underlying reason that still needs to be investigated.